docs/content/preview/yugabyte-platform/security/authentication/oidc-authentication-jumpcloud.md

```diff
@@ -0,0 +1,192 @@
+
+## Create an application in JumpCloud
+
+To use JumpCloud for your IdP, do the following:
+
+1. Sign in to JumpCloud using an administrator account.
+
+1. Create an application.
+
+    - Under **SSO Applications**, click **Add New Application**.
+    - Select **Custom Application**, and make sure the integration supports "SSO with OIDC" on the next page.
+    - Under **Manage Single Sign-On (SSO)**, select **Configure SSO with OIDC**, and click **Next**.
+    - Under **Enter General Info**, add the application name (for **Display Label**), **Description**, and logo (for **User Portal Image**), and select **Show this application in User Portal**.
+
+      This information is displayed as a tile when users sign in to YugabyteDB Aeon.
+
+    - Click **Configure Application**.
+
+1. Configure your application.
+
+    Under **SSO > Endpoint Configuration**, configure the following:
+
+    - **Redirect URIs** - enter `https://<your-YugabyteDB-Anywhere-IP-address>/api/v1/callback?client_name=OidcClient`.
+    - **Client Authentication Type** - select **Client Secret Post**.
+    - **Login URL** - enter `https://<your-YugabyteDB-Anywhere-IP-address>/login`.
+
+    Under **Attribute Mapping**, for **Standard Scopes**, select **Email** and **Profile**.
+
+    Click **Activate** when you are done.
+
+    You will be prompted in a pop up to save the **Client ID** and **Client Secret**. Save these in a secure location, you will need to provide these credentials in YugabyteDB Anywhere.
+
+1. Configure Attributes and Identity Management as required.
+
+1. Integrate the user in JumpCloud.
+
+    - Navigate to **User Groups**, select the user groups you want to access YugabyteDB Aeon, and click **Save** when you are done.
+
+To configure JumpCloud federated authentication in YugabyteDB Aeon, you need the following application properties:
+
+- **Client ID** and **Client Secret** of the application you created. These are the credentials you saved when you activated your application. The **Client ID** is also displayed on the **SSO** tab.
+
+For more information, refer to the [JumpCloud](https://jumpcloud.com/support/sso-with-oidc) documentation.
+
+### Configure YugabyteDB Anywhere
+
+#### Enable OIDC authentication
+
+To configure User authentication in YugabyteDB Anywhere, do the following:
+
+1. Navigate to **Admin > Access Management > User Authentication** and select **ODIC configuration**.
+1. Under **OIDC configuration**,  configure the following:
+
+    - **Client ID** and **Client Secret** - enter the client ID and secret of the JumpCloud application you created.
+    - **Discovery URL** - enter `https://oauth.id.jumpcloud.com/.well-known/openid-configuration`.
+    - **Scope** - enter `openid email`.
+    - **Email attribute** - enter your registered email.
+
+1. Click **Save**.
+
+You are redirected to sign in to your IdP to test the connection. After the test connection is successful, OIDC authentication is enabled.
+
+### Configure a universe
```

Comment:
Are these section that follow identical to Azure?

Reply:
they are identical with minor modifications.